stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/security/KeyStore.java

```diff
@@ -0,0 +1,244 @@
+     *
+     * @return certificate corresponding to the alias.
+     * @throws KeyStoreException
+     */
+    public Certificate getDefaultCertificate() throws KeyStoreException {
+        String keyAlias = ConfigRegistry.getInstance().getInstanceConfigValue(KEY_STORE_CONFIG, KEY_ALIAS);
+        return getCertificate(keyAlias);
+    }
+
+    private void loadKeyStore() {
+        ConfigRegistry configRegistry = ConfigRegistry.getInstance();
+        String keyStoreLocation = configRegistry.getInstanceConfigValue(KEY_STORE_CONFIG, KEY_STORE_LOCATION);
+        if (keyStoreLocation != null) {
+            char[] keyStorePassword = configRegistry.getInstanceConfigValue(
+                    KEY_STORE_CONFIG, KEY_STORE_PASSWORD).toCharArray();
+            String keystoreType = configRegistry.getInstanceConfigValue(KEY_STORE_CONFIG, KEY_STORE_TYPE);
+            try (InputStream file = new FileInputStream(new File(keyStoreLocation))) {
+                keyStore = java.security.KeyStore.getInstance(keystoreType);
+                keyStore.load(file, keyStorePassword);
+            } catch (FileNotFoundException e) {
+                throw new BallerinaException("Failed to load keystore: file not found: " + keyStoreLocation, e);
+            } catch (Exception e) {
+                throw new BallerinaException("Failed to load keystore: " + e.getMessage(), e);
+            }
+        } else {
+            try {
+                log.warn("KeyStore is not configured");
+                keyStore = java.security.KeyStore.getInstance("JKS");
+            } catch (java.security.KeyStoreException e) {
+                throw new BallerinaException("Failed to instantiate keystore: ", e);
+            }
+        }
+    }
+
+    private void loadTrustStore() {
+        ConfigRegistry configRegistry = ConfigRegistry.getInstance();
+        String trustStoreLocation = configRegistry.getInstanceConfigValue(TRUST_STORE_CONFIG, TRUST_STORE_LOCATION);
+        if (trustStoreLocation != null) {
+            char[] trustStorePassword = configRegistry.getInstanceConfigValue(TRUST_STORE_CONFIG, TRUST_STORE_PASSWORD)
+                    .toCharArray();
+            String trustStoreType = configRegistry.getInstanceConfigValue(TRUST_STORE_CONFIG, TRUST_STORE_TYPE);
+            try (InputStream file = new FileInputStream(new File(trustStoreLocation))) {
+                trustStore = java.security.KeyStore.getInstance(trustStoreType);
+                trustStore.load(file, trustStorePassword);
+            } catch (FileNotFoundException e) {
+                throw new BallerinaException("Failed to load trustStore: file not found: " + trustStoreLocation, e);
+            } catch (Exception e) {
+                throw new BallerinaException("Failed to load trustStore: " + e.getMessage(), e);
+            }
+        } else {
+            try {
+                log.warn("TrustStore is not configured");
+                trustStore = java.security.KeyStore.getInstance("JKS");
```

Comment:
This is like a default case right? Any particular reason to use JKS in a default case instead of PKCS12?